libs/application-generic/src/logging/index.ts

```diff
@@ -10,135 +11,85 @@ export function getErrorInterceptor(): NestInterceptor {
 }
 export { Logger, LoggerModule, PinoLogger, storage, Store, getLoggerToken };
 
-const loggingLevelArr = ['error', 'warn', 'info', 'verbose', 'debug'];
-
 const loggingLevelSet = {
-  error: 50,
-  warn: 40,
+  trace: 10,
+  debug: 20,
   info: 30,
-  verbose: 20,
-  debug: 10,
+  warn: 40,
+  error: 50,
+  fatal: 60,
+  none: 70,
 };
-
-interface ILoggingVariables {
-  env: string;
-  level: string;
-
-  hostingPlatform: string;
-  tenant: string;
-}
+const loggingLevelArr = Object.keys(loggingLevelSet);
 
 export function getLogLevel() {
-  let logLevel = process.env.LOGGING_LEVEL ?? 'info';
-
-  if (loggingLevelArr.indexOf(logLevel) === -1) {
-    // eslint-disable-next-line no-console
-    console.log(`${logLevel}is not a valid log level of ${loggingLevelArr}. Reverting to info.`);
+  let logLevel = null;
 
+  if (process.env.LOGGING_LEVEL || process.env.LOG_LEVEL) {
+    logLevel = process.env.LOGGING_LEVEL || process.env.LOG_LEVEL;
```

Comment:
shouldn't this be reversed? like
```suggestion
    logLevel = process.env.LOG_LEVEL || process.env.LOGGING_LEVEL;
```

Reply:
its not about priority here, so any value should be fine, it's meant to be only kept for some time to support backward the old environment variable.